core/proto/keytransparency_v1_grpc/keytransparency_v1_admin_grpc.proto

```diff
@@ -87,3 +73,32 @@ service KeyTransparencyAdminService {
     };
   }
 }
+
+// The AccountRecoveryService API holds a set of account recovery keys.  
+// Access to these APIS are logged and restricted to properly authenticated
+// and authorized callers.  Only accounts that have put one of the recovery
+// keys in their set of authorized_keys can be administered.  Third parties can
+// implement this API in order to provide trust agility with regard to account
+// reset.
+// 
+// Key Rotation: 
+// The set of account recovery keys may rotate over time with new keys being
+// added every X months, X * N months before they become the currently active signing
+// key, and another Y months before they are deactivated from the list of valid
+// verifying keys. In order to maintain the ability of this service to
+// administer accounts, they must refresh their set of authorized public keys,
+// ideally every X months.
+service AccountRecoveryService {
+
+  // GetAuthorizedKeys returns the public keys associated with this service, and the currently active key.
+  rpc GetAutorizedKeys (keytransparency.v1.proto.GetAuthorizedKeysRequest) returns (keytransparency.v1.proto.GetAuthorizedKeysResponse) {}
```

Comment:
I'm unsure of how much adherence you want to https://cloud.google.com/apis/design/, but here it goes:

At a first glance, usually Get methods work on an entity, so the expected return of `GetAuthorizedKeys` would be an `AuthorizedKey` entity, instead of `GetAuthorizedKeysResponse`.

There's an underlying question of which entities you have and how you model them. Some alternatives would be to implement this as a view on GetAccount (which returns the keys as part of the account, provided the correct input) or have AuthorizedKeys as their own collection, possibly modelled as /accounts/$account_id/authorizedkeys/$key_id/. It's hard to say just within the context of this review, so you may want to have a broader API discussion first, outside of Github.